documentation/library_guide/introduction/onedpl_gsg.rst

```diff
@@ -44,6 +44,44 @@ All |onedpl_short| header files are in the ``oneapi/dpl`` directory. Use ``#incl
 To use tested C++ standard APIs, you need to include the corresponding C++ standard header files
 and use the ``std`` namespace.
 
+CMake Support
+-------------
+`CMake <https://cmake.org/cmake/help/latest/index.html>`_ generates build scripts which can then be used to build and link your application.  |onedpl_short| can be added to your project via CMake.  
+
+A simple example for Linux is provided below.  For more detailed usage and options, please look to the `CMake Support Page <https://www.intel.com/content/www/us/en/docs/onedpl/developer-guide/current/cmake-support.html>`_.
+
+Simple Example CMake File
+*************************
+To use |onedpl_short| with CMake, create a CMakeLists.txt file for your project's base directory and use `find_package <https://cmake.org/cmake/help/latest/command/find_package.html>`_ and `target_link_libraries <https://cmake.org/cmake/help/latest/command/target_link_libraries.html>`_ to add oneDPL.
+For example:
+
+.. code:: cpp
+

```

Comment:
What do you think about adding:
```
  if (CMAKE_HOST_WIN32)
    find_package(oneDPLWindowsIntelLLVM)
  endif()
```
So the example can seamlessly work on both Windows and Linux?

Reply:
I'm somewhat indifferent about this.  I think its fine to include, but that alone wont make the full example work for Windows because the invocation would need to use the ninja generator in the invocation example.  

I expect that we want to keep just a single command line invocation example on this page, and not have to explain that option here.  

I've added it, but the example still is described as a "simple example for Linux" because we don't have a command line invocation which will work for Windows.  However, if someone quickly copies this CMakeLists.txt file it should work for either platform.

Reply:
> However, if someone quickly copies this CMakeLists.txt file it should work for either platform.
Does it require a certain version of cmake? Which scenario should it work as is?

Reply:
It should have the same CMake version requirements as oneDPL generally.  
On Linux, it should not impose any extra version requirements.  On Windows, it will complain appropriately if below version 3.20.
So Linux 3.11+ and Windows 3.20+ should work as is with a supported compiler.

Reply:
Whoops!  I think I misunderstood your question.  I thought you were asking if the proposed addition of `find_package(oneDPLWindowsIntelLLVM)` imposed any extra CMake version requirements (say on the Linux side of things).  I was answering that question.  `find_package(oneDPLWindowsIntelLLVM)` does not impose any additional CMake version requirements when used. 

I think your original question was about the simple example prior **without** `find_package(oneDPLWindowsIntelLLVM)`.  
Without the addition  of `find_package(oneDPLWindowsIntelLLVM)`, it should work on Linux with any CMake version 3.11+ and a supported compiler.  On Windows it should work on CMake version 3.26+ with icx[-cl].  Windows + icpx will not work, and icx[-cl] will not work prior to 3.26.  

I have not yet tried the newest version CMake 3.27, so I don't know yet if that has any changes for better or worse.

Reply:
Also, perhaps it is worth noting that the   `if (CMAKE_HOST_WIN32) ... endif()` wrapper around `find_package(oneDPLWindowsIntelLLVM)` is not functionally necessary.  oneDPLWindowsIntelLLVM checks `CMAKE_HOST_WIN32` internally, and should not run Linux builds which include it.

I still think it is good to have there effectively as a functional comment to show it is only required on Windows.

Reply:
I've removed the `find_package(oneDPLWindowsIntelLLVM)` lines, and added a reference to Windows details before the link to the separate CMake Support page.  I think this may be best.  Let me know if you disagree.

Reply:
> Also, perhaps it is worth noting that the if (CMAKE_HOST_WIN32) ... endif() wrapper around find_package(oneDPLWindowsIntelLLVM) is not functionally necessary. oneDPLWindowsIntelLLVM checks CMAKE_HOST_WIN32 internally, and should not run Linux builds which include it.

We can do, if we ship `oneDPLWindowsIntelLLVM` for both systems. Let me check it.

Reply:
> We can do it if we ship oneDPLWindowsIntelLLVM for both systems. Let me check it.

Yes, we provide `oneDPLWindowsIntelLLVM` on Linux as well. But I am not sure if it will be so in the future because it is not straightforward. So it might be better to be more verbose with `(CMAKE_HOST_WIN32) ... endif()`

Reply:
> I've removed the find_package(oneDPLWindowsIntelLLVM) lines, and added a reference to Windows details before the link to the separate CMake Support page. I think this may be best. Let me know if you disagree.

I agree, thanks

Reply:
> Yes, we provide `oneDPLWindowsIntelLLVM` on Linux as well. 

Thanks for checking.  I thought this was the case.